Code/Kernel/Sources/keyboard.c

```diff
@@ -1,4 +1,5 @@
 #include "../Includes/keyboard.h"
+#include "../Includes/shell.h"
```

Comment:
No need, it was included in `keyboard.h` so you can remove it.

Reply:
Thank you for your patient guidance. I have removed it